docs/Documentation/Development/Containers/index.md

```diff
@@ -5,37 +5,141 @@ parent: Containers
+
+
+## Building software images
+Regardless of the runtime platform, images are built from a special configuration file. A `Dockerfile` is such a configuration for Docker, while Apptainer uses a "Definition File" (with a `.def` extension). These files specify the installation routines necessary to create the desired application, as well as any additional software packages to install and configure in this environment that may be required. You can think of these files as "recipes" for installing a given application you wish to containerize.
+
+Building Docker or Singularity images requires root/admin privileges and cannot be done directly by users of HPC systems. Docker is available on most platforms, and users with admin privileges on a local machine (such as your laptop) can build Docker images locally. The Docker image file can then be pushed to a registry and pulled on the HPC system using Singularity as described [here](registries.md), or a tool such as [Docker2Singularity](https://github.com/singularityhub/docker2singularity) may be used to convert the image to a Singularity format. Alternatively, users with admin privileges on a Linux system can run Singularity locally to build images. Another option is to use [Singularity Container Service](https://cloud.sylabs.io/), which provides free accounts with a limited amount of container build time.
 
-## HPC hardware
-Both Singularity and Docker provide the ability to use hardware based features on the HPC systems such as GPUs. A common usage for containers is packaging of GPU enabled tools such as TensorFlow. Singularity enables access to the GPU and driver on the host. In principle, the MPI installations can be also be accessed from correctly configured containers, but care is needed to ensure compatibility between the libraries on the host and container.
+### Example Docker build workflow for HPC users
+
+Because of the permission limitations described above, it is recommended that HPC users start with building a Docker image locally, e.g., on your laptop. If you plan to regularly containerize applications, you can request Docker to be installed at the admin-level on your work computer from the [IT Service Portal](https://nrel.servicenowservices.com). This section will describe a simple workflow for building a Docker image locally, exporting it as a `.tar`file, uploading it to Kestrel, and converting it to an Apptainer image for execution on HPC.
```

Comment:
Since non-NREL users will also be reading this, should mention that only NREL users can request this from ITS, or make the directions more general. We frequently get questions from external users about why they can't access ITS links, so want to avoid confusion.